src/console_cmds.cpp

```diff
@@ -614,6 +615,118 @@ DEF_CONSOLE_CMD(ConBanList)
 	return true;
 }
 
+/**
+* Loads the ban list from the specified path.
+* @param path The path to load the ban list from.
+* @return The number of entries loaded from the file or -1 if an error occurred while loading entries.
+*/
+static int LoadBanList(const std::string& path)
+{
+	const char* const group_names[] = {
+		"bans",
+		nullptr
+	};
+
+	const auto ini = new IniFile(group_names);
+	ini->LoadFromDisk(path, NO_DIRECTORY);
+	const auto ban_group = ini->GetGroup("bans", false);
+
+	auto loaded_count = 0;
+	auto is_loaded = ban_group != nullptr;
```

Comment:
Code style we prefer variables have an explicit type.
```suggestion
	int loaded_count = 0;
	bool is_loaded = ban_group != nullptr;
```

Reply:
Excellent. I did not see that mentioned on the [Code Style wiki page](https://wiki.openttd.org/en/Development/Coding%20style) or in the CONTRIBUTING.md file. On a related note, we may want to update the [EN](https://wiki.openttd.org/en/Archive/Source/OpenTTDDevBlackBook/IConsole/AddFunctions) and PL wiki pages on adding a console function as they appear to be very out of date.